src/main/java/io/stargate/sgv2/jsonapi/config/constants/HttpConstants.java

```diff
@@ -15,9 +15,27 @@ public interface HttpConstants {
   /** JSON API Embedding serive Authentication token header name. */
   String EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME = "x-embedding-api-key";
 
+  /** JSON API Embedding serive access id header name. */
+  String EMBEDDING_AUTHENTICATION_ACCESS_ID_HEADER_NAME = "x-embedding-access-Id";
```

Comment:
Although header names are case-sensitive, I think recommendation is to use all lower-case, so:

   x-embedding-access-id

(f.ex https://scrapfly.io/blog/what-case-should-http-headers-be/)